packages/browser/tests/redux/backtraceReduxMiddlewareTests.spec.ts

```diff
@@ -0,0 +1,189 @@
+import { BacktraceBreadcrumbs } from '@backtrace/sdk-core/src';
+import { BacktraceClient } from '../../src/BacktraceClient';
+import { createBacktraceReduxMiddleware } from '../../src/redux/BacktraceReduxMiddleware';
+import { configureStore, createSlice, PayloadAction, Action, Middleware } from '@reduxjs/toolkit';
+
+const clientBreadcrumbsEnabled = BacktraceClient.initialize({
+    name: 'test-cleint',
+    version: '1.2.3',
+    url: 'https://test-bt-client-url.sp.backtrace.io',
+    breadcrumbs: {
+        enable: true,
+    },
+});
+
+interface TestState {
+    testArr: string[];
+    testBool: boolean;
+}
+
+const initialState: TestState = {
+    testArr: [],
+    testBool: false,
+};
+
+const internalError = new Error('Test internal error');
+
+const testSlice = createSlice({
+    name: 'test',
+    initialState,
+    reducers: {
+        addToTestArray(state, action: PayloadAction<string>) {
+            state.testArr.push(action.payload);
+        },
+        toggleTestBool(state, action: PayloadAction<undefined>) {
```

Comment:
The store variable is unused. If for any reason we have to keep it, can we add a linter ignore?

Reply:
Fixed